website/docs/guides/composedb-client/user-sessions.mdx

```diff
@@ -0,0 +1,248 @@
+compose.setDID(session.did)
+```
+
+:::tip
+Additional chain support is continually being added. You can find the link to each chain and its docs below.
+
+- [Tezos](https://did.js.org/docs/api/modules/pkh_tezos)
+:::
+
+## Scopes
+In the last line of the above examples, you see a `resources` array. This is effectively a scope of permission that the user is assigning. In ComposeDB, these resources are the models you’ve included in your composite.
+
+The compose client offers a simple getter `compose.resources` that formats all model streamIDs in your composite for did-session. You can then pass this as a configuration option. 
+
+The `compose.resources` is an array of URI-formatted streamIDs of models, for example:
+
+```jsx
+[ "ceramic://*?model=kjzl6hvfrbw6c7keo17n66rxyo21nqqaa9lh491jz16od43nokz7ksfcvzi6bwc", "ceramic://*?model=kjzl6hvfrbw6c99mdfpjx1z3fue7sesgua6gsl1vu97229lq56344zu9bawnf96"]
+```
+
+## Additional Usage
+Once you authorize and create a session, you can serialize and store session for later. Sessions by default are valid for 7 days. 
+
+```jsx
+// Create session as above, store for later
+const session = await DIDSession.authorize(authMethod, { resources: [...]})
+const sessionString = session.serialize()
+
+// write/save session string where you want (ie localstorage)
+// ...
+
+// Later re initialize session
+const session2 = await DIDSession.fromSession(sessionString)
+const compose = new ComposeClient()
+compose.did = session.did
+```
+
+### Session Lifecycle
+
+Additional helper functions are available to help you manage a session lifecycle and the user experience.
+
+```jsx
+// Check if authorized or created from existing session string
+didsession.hasSession
+
+// Check if session expired
+didsession.isExpired
+
+// Get resources session is authorized for
+didsession.authorizations
+
+// Check number of seconds till expiration, may want to re auth user at a time before expiration
+didsession.expiresInSecs
+```
+
+### Complete Example
+
+A typical usage pattern is to store sessions in localstorage, when a user loads your app you can load an existing session or create a new one. When you start making mutations with the client instance, you should make sure that the session is not expired. 
+
+```jsx
+import { DIDSession } from 'did-session'
+import type { AuthMethod } from '@didtools/cacao'
+import { EthereumWebAuth, getAccountId } from '@didtools/pkh-ethereum'
+import { ComposeClient }from '@composedb/client'
+
+const ethProvider = // import/get your web3 eth provider
+const addresses = await ethProvider.request({ method: 'eth_requestAccounts' })
+const accountId = await getAccountId(ethProvider, addresses[0])
+const authMethod = await EthereumWebAuth.getAuthMethod(ethProvider, accountId)
+
+const compose = new ComposeClient()
+
+const loadSession = async (authMethod) => {
+	// Check if user session already in storage 
+  const sessionStr = localStorage.getItem('didsession')
```

Comment:
Could you add warnings about using localstorage as in the DID docs please? This shouldn't be treated as a secure option.

Reply:
i didn't see those warnings in the DID docs, what should we put there?

Reply:
The warning is on the page this example seem to be copied from, see https://did.js.org/docs/guides/using-with-composedb-client